crates/dbs-virtio-devices/src/vsock/muxer/muxer_impl.rs

```diff
@@ -463,12 +507,13 @@ impl VsockMuxer {
     }
 
     /// Parse a host "connect" command, and extract the destination vsock port.
-    fn read_local_stream_port(stream: &mut Box<dyn VsockStream>) -> Result<u32> {
+    fn read_local_stream_port(stream: &mut Box<dyn VsockStream>) -> Result<Option<u32>> {
```

Comment:
pls make the return value more readable, like using Enum to distinguish passfd or normal mode